.rhcicd/floorist/floorplan.yaml

```diff
@@ -165,6 +165,57 @@ objects:
             a.display_name,
             et.display_name,
             es.subscription_type;
+      # Count the number of events grouped by bundle, application and status.
+      - prefix: insights/notifications/events_by_bundle_app_status
```

Comment:
If an event triggers an email and a webhook action, it will be counted twice, is it the expected behaviour ?

Reply:
I was considering counting all the successful, unsuccessful and whatever deliveries, yes, because then we need to perform another query to just count the incoming events.

Like, this one counts every delivery, and the other one will count the volume of the incoming events per app, so that they can compare both.

Thank you for your comment and the review :+1:

Reply:
Ok fine for me then